src/iop/colorbalance.c

```diff
@@ -87,6 +87,17 @@ typedef enum _controls_t
   BOTH
 } _controls_t;
 
+typedef enum dt_iop_colorbalance_pickcolor_type_t {
```

Comment:
Please { on next line.

Reply:
This is clang-format, on enums it works like this.

Reply:
Yes, I know but that's not dt style (and that's not a style I have chosen, I wasn't dev at the start of the project, but consistency is important). Will fix :)